### PR TITLE
Update README for /allStops

### PR DESCRIPTION
**Pull request by kevinchan159**
this for you @AAAstorga

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,6 +82,7 @@ The date, time, and daysOfWeek fields can specify an alert that takes place for
 | name     | String   | The name of the bus stop.                 |
 | lat      | Double   | The latitude coordinate of the bus stop.  |
 | long     | Double   | The longitude coordinate of the bus stop. |
+| type     | String   | This is just the string "busStop".        |
```
Comment (AAAstorga) [inline README.md]: why is this here then? We need more explanation please

Comment (kevinchan159) [reply README.md]: omfg
